src/docs/antora/modules/ROOT/pages/events.adoc

```diff
@@ -211,6 +211,16 @@ Automatic re-publication of the events can be enabled via the xref:appendix.adoc
 .The transactional event listener arrangement after execution
 image::event-publication-registry-end.png[]
 
+[[publication-registry.event-lifecycle]]
+=== The Event Lifecycle
+
+Events can be in one of two states:
+
+* Incomplete
+* Complete
+
+Events remain incomplete until successfully delivered. That means an event can be incomplete because it has yet to be delivered to the respective listener, the listener is currently processing it, or a previous attempt to deliver it has failed, for example, because the listener has thrown an exception. There is currently no mechanism to detect which of those cases occurred and how many times.
+
 [[publication-registry.starters]]
 === Spring Boot Event Registry Starters
```

Comment:
That should be `Spring Modulith Event Registry Starters`, shouldn't it? Otherwise, I'm unsure what the relation to Spring Boot is.